superbench/analyzer/rule_base.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A base module for rule-related module."""
+
+import re
+
+from superbench.common.utils import logger
+from superbench.analyzer import file_handler
+
+
+class RuleBase():
+    """RuleBase class."""
+    def __init__(self):
+        """Init function."""
+        self._sb_rules = {}
+        self._benchmark_metrics_dict = {}
+
+    def _get_metrics_by_benchmarks(self, metrics_list):
+        """Get mappings of benchmarks:metrics of metrics_list.
```

Comment:
`Get mappings of benchmarks:metrics of metrics_list.` => `Get mappings of benchmarks:metrics from metrics_list.`

Reply:
thanks, updated